libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioTimestampPoller.java

```diff
@@ -272,7 +261,7 @@ private void updateState(@State int state) {
     }
   }
 
-  @RequiresApi(19)
+  // DO NOT SUBMIT in-line this
   private static final class AudioTimestampV19 {
```

Comment:
Could we rename this to `AudioTimestampWrapper`, or something that is less API-specific?
```suggestion
  private static final class AudioTimestampWrapper {
```

Reply:
Good idea, done